.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Question
+about: ask a question about the behavior of chproxy or the documentation
+title: '[QUESTION] <title>'
+labels: ''
+assignees: ''
+
+---
```

Comment:
It's not enough. This will just create an additional item in the list along:
- Bug Report
- Feature Request
- Report a security vulnerability
as **Question**

IMO we could provide a template that can indicate what kind of information may be useful to submit while posing question. 
e.g. 

**Which feature your question relates to?**
E.g. caching

**Question details**
A clear and concise content of the question.

**Additional context/motivations**
Is there any info or motivation to be shared? E.g. asking the question because started to investigate if chproxy's feature X is suitable for my use case.